docs/fundamentals/snippets/custom-urls/AspireApp.AppHost/Program.WithUrl.cs

```diff
@@ -0,0 +1,18 @@
+﻿internal static partial class Program
+{
+    internal static void WithUrlExample(string[] args)
+    {
+        // <withurl>
+        var builder = DistributedApplication.CreateBuilder(args);
+
+        var api = builder.AddProject<Projects.AspireApp_Api>("api");
+
+        var adminUrl = ReferenceExpression.Create(
+            $"{api.GetEndpoint("https")}/admin");
+
+        api.WithUrl(adminUrl, "Admin Portal");
```

Comment:
You can just pass the interpolated string directly to `WithUrl`, no need to manually call `ReferenceExpression.Create`